bender-sim.mk

```diff
@@ -13,4 +13,5 @@ ifeq ($(TECH_SIM), 1)
 	sim_targs += -t tech_sim
 	sim_defs  += -D INITIALIZE_MEMORY
 	sim_defs  += -D INITIALIZE_OUTPUT
+	sim_defs  += -D GF12_FLL
```

Comment:
What is this define for?

Reply:
I got it, probably this define is also needed in case we do synthesis (otherwise during synthesis the technological FLL is by default excluded).

Reply:
What is the flag used to indicate synthesis? A la TECH_SIM=1

Reply:
[It gets automatically exported ](https://github.com/pulp-platform/astral/blob/astral/bender-synth.mk#L10)